Add changelog and update Release.toml for v1.0.6 (#1348)
bottlerocket-os/bottlerocket: CHANGELOG.md

# v1.0.6 (2021-02-26)

## OS Changes

> * Add metricdog support to send anonoymous bottlerocket metrics ([#1006], [#1322])

Reviewer: ```suggestion
* Add metricdog to support sending anonymous metrics ([#1006], [#1322])
```

* Add a vmware-dev variant ([#1292], [#1288], [#1290])

> * Add k8s static pods support ([#1317])

Reviewer: ```suggestion
* Add Kubernetes static pods support ([#1317])
```

> * Add high-level 'set' subcommand for changing settings in apiclient ([#1278])

Reviewer: ```suggestion
* Add high-level 'set' subcommand for changing settings using apiclient ([#1278])
```

> * Add "shibaken" generator to populate user-data with public keys from IMDS ([#1331])

Reviewer: ```suggestion
* Allow admin container to use SSH public keys from user data ([#1331], [#19](https://github.com/bottlerocket-os/bottlerocket-admin-container/pull/19))
```

* Add support for kubelet in standalone mode and TLS auth ([#1338])
* Add https-proxy and no-proxy settings to updog ([#1324])

> * Add support for ECR Public to hostctr ([#1296])

Reviewer: ```suggestion
* Add support for pulling host-containers from ECR Public ([#1296])
```

> * Add network proxy support to k8s 1.19 ([#1337])

Reviewer: ```suggestion
* Add network proxy support to aws-k8s-1.19 ([#1337])
```

> * Add support for kmot kit ([#1287], [#1286], [#1285])

Author: Does this seem ok or sounds incomplete ?

Reviewer: (typo: "kmod")

Maybe "Add support for kmod kits to ease building of third-party kernel modules"?

Reviewer: Also, I think this can go under build changes.

> * Modify default label for containers ([#1318])

Reviewer: ```suggestion
* Modify default SELinux label for containers to align with upstream ([#1318])
```

> * Switch from TermLogger to SimpleLogger ([#1282], **thanks @hencrice!**)

Reviewer: This can go under build changes, I think, it doesn't affect runtime.

> * Allow overriding the "pretty" name of the OS inside the image ([#1330])

Reviewer: This can go under build changes.

> * Add aliases for container-selinux types ([#1316])

Reviewer: ```suggestion
* Add aliases for container-selinux types to align with community ([#1316])
```

> * Declare dependencies on model and defaults files in storewolf ([#1319])

Author: Should it be: ```storewolf: Declare dependencies on model and defaults files ``` ?

Reviewer: Also, this can go under build changes.

> * Refactor default settings files to allow sharing in storewolf ([#1303])

Reviewer: ```suggestion
* Refactor default settings files to allow sharing in storewolf ([#1303], [#1329])
```

Same "storewolf: " prefix as the prior line, if we do that.

Also, this can go under build changes.

> * Improve models symlink logic ([#1329])

Reviewer: ```suggestion
```
This isn't a useful change on its own, it can be combined with 1303.

> * Add the ability to output vmdk via qemu-img ([#1289])

Reviewer: This can go under build changes.

> * Migrates default version of admin container from v0.5.2 to v0.6.0 ([#1347])

Reviewer: ```suggestion
* Update default versions of admin and control containers ([#1347], [#1344])
```

And add https://github.com/bottlerocket-os/bottlerocket/pull/1344 to the link list below.

## Build Changes

* Fix invalid symlinks when the BUILDSYS_NAME variable is set ([#1312])

> * Add script runner bash to link-variant ([#1323])

Reviewer: ```suggestion
* Specify bash in link-variant task for use of bash features ([#1323])
```

# v1.0.6 (2021-02-26)

> ## OS Changes

Reviewer: nit: for consistency, let's have a blank line under each heading, like past releases.

* Add support for kubelet in standalone mode and TLS auth ([#1338])
* Add https-proxy and no-proxy settings to updog ([#1324])
* Add support for pulling host-containers from ECR Public ([#1296])
* Add network proxy support to aws-k8s-1.19 ([#1337])
* Modify default SELinux label for containers to align with upstream ([#1318])
* Add aliases for container-selinux types to align with community ([#1316])
* Update default versions of admin and control containers ([#1347], [#1344])

## Build Changes

* Add the ability to output vmdk via qemu-img ([#1289])
* Add support for kmod kits to ease building of third-party kernel modules ([#1287], [#1286], [#1285])
* storewolf: Declare dependencies on model and defaults files ([#1319])
* storewolf: Refactor default settings files to allow sharing ([#1303], [#1329])
* Switch from TermLogger to SimpleLogger ([#1282], **thanks @hencrice!**)
* Allow overriding the "pretty" name of the OS inside the image ([#1330])
* Specify bash in link-variant task for use of bash features ([#1323])
* Fix invalid symlinks when the BUILDSYS_NAME variable is set ([#1312])
* Track and clean output files for builds ([#1291])

> * Update third-party software packages ([#1340], [#1336], [#1334], [#1333], [#1335], [#1190], [#1265], [#1315])

Reviewer 2: We can add #1352

Author: oh, its there in the recent force pushes, I think this one you are looking at is outdated ?
